datafusion/core/src/physical_plan/aggregates/bounded_aggregate_stream.rs

```diff
@@ -31,9 +31,9 @@ use futures::stream::{Stream, StreamExt};
 use hashbrown::raw::RawTable;
 use itertools::izip;
 
-use crate::execution::context::TaskContext;
-use crate::execution::memory_pool::proxy::{RawTableAllocExt, VecAllocExt};
-use crate::execution::memory_pool::{MemoryConsumer, MemoryReservation};
+use datafusion_execution::TaskContext;
```

Comment:
this is the core of the change -- use the actual dependencies rather than backward compatible pub use's